README.md

```diff
@@ -19,7 +19,7 @@ Aka IFTTT orchestration for Ops.
 > In order to access StackStorm Web UI, please don't forget to ensure that http/https ports are opened in your firewall system.
 
 ## Requirements
-At least 2GB of memory and 3.5GB of disk space is required, since StackStorm is shipped with RabbitMQ, PostgreSQL, Mongo, nginx and OpenStack Mistral.
```

Comment:
Do we want to update the system requirements now we've removed postgreSQL and mistral?

Reply:
Existing requirements should be somewhat OKish. For example, https://docs.stackstorm.com/install/system_requirements.html recommends similar system resources just for the testing environment.